src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -69,10 +69,11 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
                 psi.Arguments = testConsoleAppArgs;
             }
 
+            var p = new Process() { StartInfo = psi };
             // Return the handle to the process, which may or not be started
             return new RemoteInvokeHandle(options.Start ?
                 Process.Start(psi) :
-                new Process() { StartInfo = psi }, options,
+                p, options, 0 /* I cannot provide p.ExitCode because p has to exit first. Therefore this property is not supported for non uap and would throw exception when used. */,
```

Comment:
I cannot provide p.ExitCode because p has to exit first. Therefore this property is not supported for non uap and would throw exception when used.